package.json

```diff
@@ -52,6 +53,7 @@
     "npm-run-all": "^4.1.5",
     "prettier": "^3.1.0",
     "tsx": "^4.6.0",
+    "ubiquibot-logger": "^0.3.5",
```

Comment:
https://www.npmjs.com/package/@ubiquity-dao/ubiquibot-logger should be the proper package.